search_events_app/services/filter_manager.py

```diff
@@ -1,13 +1,14 @@
 from search_events_app.dto.dto_filter import DTOFilter
 from search_events_app.services.filters.country_filter import CountryFilter
+from search_events_app.services.filters.feature_filter import FeatureFilterManager
 from search_events_app.services.filters.language_filter import LanguageFilter
 from search_events_app.services.filters.online_filter import OnlineFilter
```

Comment:
One thing that you can do to avoid importing every filter is:
in the file: `search-events/search_events_app/services/filters/__init__.py` import every filter that you need and then in the rest of the project you can just import:
`from search_events_app.services import filters`
and there you can access to all the filters

Reply:
Resolved